server/routes.go

```diff
@@ -36,14 +36,10 @@ func cacheDir() string {
 }
 
 func generate(c *gin.Context) {
-	var req api.GenerateRequest
-	if req.ModelOptions == nil {
-		req.ModelOptions = &api.DefaultModelOptions
+	req := api.GenerateRequest{
```

Comment:
This was a bug that I fixed in main today, the options should be a pointer. This code is just overriding whatever options that get passed in with the defaults.

Reply:
can you elaborate? from what I understand, the values will override defaults if they're non-zero values

Reply:
ah, I mis-read that, you're right